<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1211</width>
    <height>906</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>841</height>
     </rect>
    </property>
    <property name="title">
     <string>Input Parameters</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>341</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>General parameters</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>31</y>
        <width>321</width>
        <height>194</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Power in the cell, P:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pwrLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>System bandwidth, B:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="bwLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Total subcarriers, Cs:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="totScLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Average data rate of each subcarrier, R:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="drLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Number of channel gain intervals, M:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="snrsLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Max. subcarriers in each call, K:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="maxCallScLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxLength">
            <number>3</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>398</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Calculate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="pbRadioButton">
        <property name="text">
         <string>Blocking Probability, Pb</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="scRadioButton">
        <property name="text">
         <string>Number of subcarriers, M</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>760</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>341</width>
       <height>501</height>
      </rect>
     </property>
     <property name="title">
      <string>Traffic</string>
     </property>
     <widget class="QTableView" name="scTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>311</width>
        <height>151</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>311</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Subcarriers/call distribution:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="scDistComboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Uniform</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Binomial</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Poisson</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gamma</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Add values manually</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Mean value of subcarriers/call:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="avCallScLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Set:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="lRadioButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>arrival rate, λ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="aRadioButton">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Traffic load, α</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>arrival rate, λ:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Call duration distribution:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="callDurDistComboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Uniform</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Binomial</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Poisson</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gamma</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Add values manually</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Subcarriers of longest call:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="hMaxScLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_19">
         <item>
          <widget class="QLabel" name="lngstCallDurLabel">
           <property name="text">
            <string>Mean duration of longest call:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="hMaxLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_20">
         <item>
          <widget class="QLabel" name="label_14">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Traffic load distribution:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="traffLoadDistComboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Uniform</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Binomial</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Poisson</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gamma</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Add values manually</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Offered traffic load per subcarrier, α:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="aLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>70</y>
       <width>371</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Channel gain</string>
     </property>
     <widget class="QTableView" name="gainTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>281</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>281</width>
        <height>108</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Distribution for channel gains:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="gainDistComboBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>Uniform</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Binomial</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Poison</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Gamma</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Add values manually</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>call type (channel gain):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="gainComboBox">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_54">
         <item>
          <widget class="QLabel" name="label_17">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>channel gain, g:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="avSnrLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_55">
         <item>
          <widget class="QLabel" name="label_22">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>probability, pg:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pGainLineEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="layoutWidget_6">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>760</y>
       <width>201</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>call type (subcarriers):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="scCallTypeComboBox">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>290</y>
       <width>371</width>
       <height>461</height>
      </rect>
     </property>
     <property name="title">
      <string>Input table</string>
     </property>
     <widget class="QTableView" name="inputTableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>351</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
    </widget>
    <zorder>layoutWidget_9</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>pushButton</zorder>
    <zorder>groupBox_6</zorder>
    <zorder>groupBox_5</zorder>
    <zorder>layoutWidget_6</zorder>
    <zorder>groupBox_3</zorder>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>hLineEdit</zorder>
   <zorder>label_11</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pbRadioButton</tabstop>
  <tabstop>scRadioButton</tabstop>
  <tabstop>pwrLineEdit</tabstop>
  <tabstop>bwLineEdit</tabstop>
  <tabstop>totScLineEdit</tabstop>
  <tabstop>drLineEdit</tabstop>
  <tabstop>snrsLineEdit</tabstop>
  <tabstop>maxCallScLineEdit</tabstop>
  <tabstop>lRadioButton</tabstop>
  <tabstop>aRadioButton</tabstop>
  <tabstop>lLineEdit</tabstop>
  <tabstop>aLineEdit</tabstop>
  <tabstop>scDistComboBox</tabstop>
  <tabstop>gainDistComboBox</tabstop>
  <tabstop>avSnrLineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>inputTableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
